File: QTD2.Infrastructure/Model/DIFSurveyEmployeeResponse/ImportDatum_DIFSurveyEmployeeResponse_VM.cs
using QTD2.Infrastructure.Model.Import_CSV_VM;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.DIFSurveyEmployeeResponse
{
   public class ImportDatum_DIFSurveyEmployeeResponse_VM : ImportDatum_VM
    {
        public string EmployeeNumber { get; set; }
        public string TaskNumber { get; set; }
        public string Difficulty { get; set; }
        public string Importance { get; set; }
        public string Frequency { get; set; }
        public string NA { get; set; }
        public string Comments { get; set; }

    }
}

File: QTD.SPA/src/app/_DtoModels/DIFSurveyImport/ImportDatum_DIFSurveyEmployeeResponse_VM.ts
import { ImportDatum_VM } from "@models/ImportCSV/ImportDatum_VM";
import { ValidationError_VM } from "@models/ImportCSV/ValidationError_VM";

export class ImportDatum_DIFSurveyEmployeeResponse_VM extends ImportDatum_VM{
    employeeNumber: string;
    taskNumber: string;
    difficulty: string;
    importance: string;
    frequency: string;
    na: string;
    comments: string;
    validationErrors: ValidationError_VM[];
}